REVIEWING.md

```diff
@@ -0,0 +1,67 @@
+Reviewing Pull Requests to the Majora's Mask Decompilation Project
+=======================================================
+
+Thanks for helping us reverse engineer *The Legend of Zelda: Majora's Mask* for the N64! 
+We invite all contributors to participate in our code review program. Every review submitted helps us keep code quality high and code merged in more quickly.
+
+This document is meant to be a set of tips and guidelines for reviewers of pull requests to the project.
+For general information about the project, see [our readme](https://github.com/zeldaret/mm/blob/master/README.md).
+
+Most discussions happen on our [Discord Server](https://discord.zelda64.dev) where you are welcome to ask if you need help getting started, or if you have any questions regarding this project and other decompilation projects.
+
+Useful Links
+------------
+
+- [Discord](https://discord.zelda64.dev/) - Primary discussion platform.
+- [Trello board](https://trello.com/b/ruxw9n6m/majoras-mask-decompilation) - We use this to track decompilation progress, not GitHub Issues.
+
+- [Contributing guide](https://github.com/zeldaret/mm/blob/master/CONTRIBUTING.md) - Instructions for contributors.
+- [Installation guide](https://github.com/zeldaret/mm/blob/master/README.md#installation) - Instructions for getting this repository set up and built on your machine.
+
+- [Zelda 64 Reverse Engineering Website](https://zelda64.dev/) - Our homepage, with links to other resources.
+- [Introduction to OOT decomp](https://github.com/zeldaret/oot/blob/master/docs/tutorial/contents.md) - A very detailed tutorial on how to get started with decomp. For OOT, but largely applicable to MM as well.
+- The `#resources` channel on the Discord contains many more links
```

Comment:
I feel like this whole part feel redundant from the CONTRIBUTING file

Reply:
Agreed. By all means link to CONTRIBUTING.md, but I don't see a need to duplicate all this here, especially since any changes would also need to be duplicated.

Reply:
Would like to get some more feedback on this. Personally it doesn't bother me at all that the links are redundant.